conjure-undertow-annotations/src/main/java/com/palantir/conjure/java/undertow/annotations/Handle.java

```diff
@@ -122,6 +122,18 @@
         Class<? extends CollectionParamDecoder<?>> decoder() default DefaultParamDecoder.class;
     }
 
+    @Retention(RetentionPolicy.SOURCE)
```

Comment:
rebase + class retention (changed recently to support local codegen of published generic interfaces)